Java/FizzBuzz.java

```diff
@@ -1,19 +0,0 @@
-public class FizzBuzz

```

Comment:
You need to revert this file. Use: 
`git checkout 9d80b6b9959d3b33e200dac596d73329d161eb3c Java/FizzBuzz.java`
and then commit this modification
`git commit`

For details visit: https://stackoverflow.com/questions/2733873/reverting-a-single-file-to-a-previous-version-in-git